src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConfiguration.java

```diff
@@ -24,6 +39,17 @@
     @Value("${kafka.consumer.pollTimeout:100}")
     private long pollTimeout;
 
+    @Value("${kafka.producer.retries}")
+    private int retries;
+
+    @Bean
+    @Qualifier("chips-kafka-send")
```

Comment:
You should be able to remove the Qualifier here as we only have the one bean of type Schema

Reply:
Removed